client/blocks/inline-help/popover.jsx

```diff
@@ -95,6 +98,43 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	renderChecklistProgress = () => {
+		/**
+		 * @TODO hide on checklist page?
+		 * @TODO move to own component
+		 * @TODO close Inline Help UI on click
+		 */
+		const { checklistSummary, searchQuery, selectedSiteSlug, translate } = this.props;
+		const { numComplete, totalTasks } = checklistSummary;
+
+		if ( searchQuery || ! totalTasks || this.state.showSecondaryView ) {
```

Comment:
We should render something different (maybe just nothing) if the checklist is 100% complete.